packages/datx-jsonapi-angular/projects/datx-jsonapi-angular/src/lib/services/custom-fetch/custom-fetch.service.ts

```diff
@@ -19,7 +19,7 @@ export class CustomFetchService {
     const takeUntil$: Observable<void> | undefined = fetchOptions?.takeUntil$;
 
     const requestHeaders = {
-      ...config.defaultFetchOptions.headers,
+      ...config.defaultFetchOptions['headers'],
```

Comment:
what's the reasoning for this change?

Reply:
When I migrated to angular-cli and TS 4.3, I started getting some more TS errors and this is one of them. I had to go to many places and make this kind of a change for property access. More precisely, this is because `noPropertyAccessFromIndexSignature` is set to `true` in the tsconfig that angular-cli generates. Do you want me to disable that flag and revert the changes that needed to be done to satisfy the rule?

This is the error if I don't make the change:
```
Property 'headers' comes from an index signature, so it must be accessed with ['headers']. ts(4111)
```

Reply:
I don't know, I don't really like this syntax, and it doesn't make sense to migrate the whole project if it can be disabled.